src/RequestOptions.php

```diff
@@ -132,6 +132,14 @@ final class RequestOptions
      */
     const HTTP_ERRORS = 'http_errors';
 
+    /**
+     * idn: (bool|int, default=true) Set to false to disable exceptions
```

Comment:
Looks like this docblock needs updating.

Reply:
Thanks, fixed